src/modules/post/entity.rs

```diff
@@ -19,7 +19,7 @@ impl ToString for Scope {
     }
 }
 
-#[derive(Clone, Debug, Deserialize, Serialize, SimpleObject)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
Given that types differ between:

- Repository DTOs
- Services DTOs
- GraphQL Types

We are likely to move into defining types in each module's `graphql::types` module and deprecate the use of `entity`. My only concern right now is _Where does domain types would be implemented?_ to keep domain logic explicit I believe we must keep their implementation free of dependencies and use them in services.